opengapps-packages.mk

```diff
@@ -34,6 +33,16 @@ GAPPS_PRODUCT_PACKAGES += \
     Turbo
 endif
 
+ifneq ($(filter $(call get-allowed-api-levels),26),)
```

Comment:
@jamuir doing this because GoogleLoginService is a PRIVILEGED app and can't be override or it seems ... going to build right now